projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-block-editor-assets.php

```diff
@@ -469,6 +472,149 @@ private function setup_block_editor_screen() {
 		}
 	}
 
+	/**
+	 * Removes hooks from problematic plugins that cause errors in this endpoint.
+	 *
+	 * Some plugins conditionally load admin-only code based on is_admin(), which
+	 * returns false in REST API contexts. When these plugins hook into
+	 * enqueue_block_editor_assets without checking the context, they may call
+	 * undefined functions that were never loaded, causing fatal errors.
+	 *
+	 * This method preemptively removes hooks from known problematic plugins before
+	 * the enqueue_block_editor_assets action fires, preventing fatal errors.
+	 */
+	private function remove_problematic_plugin_hooks() {
+		global $wp_filter;
+
+		// List of problematic plugins and their callback patterns
+		$problematic_plugins = array(
+			'wpforms-lite' => array(
+				'enqueue_block_editor_assets',
+			),
+		);
+
+		foreach ( $problematic_plugins as $plugin_slug => $hook_names ) {
+			foreach ( $hook_names as $hook_name ) {
+				if ( ! isset( $wp_filter[ $hook_name ] ) ) {
+					continue;
+				}
+
+				// Iterate through all priorities for this hook
+				foreach ( $wp_filter[ $hook_name ]->callbacks as $priority => $callbacks ) {
```

Comment:
There is a lot of loops here. 

I wonder if we can cache some of this info into an array. 

Something like

Hook_callbacks_method_to_path array. Where we figure out the path of each of the callback once. 

And then we do another loop where we do the remove_action based on the plugin.